src/main/java/uk/gov/pay/api/filter/AuthorizationValidationFilter.java

```diff
@@ -31,38 +36,45 @@ public AuthorizationValidationFilter(PublicApiConfig configuration) {
     }
 
     @Override
-    public void init(FilterConfig filterConfig) {}
+    public void init(FilterConfig filterConfig) {
+    }
 
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
 
         final String authorization = ((HttpServletRequest) request).getHeader("Authorization");
 
-        if (isValidAuthorizationHeader(authorization)) {
+        if (isValidAuthorizationHeader(authorization, request)) {
             chain.doFilter(request, response);
-
         } else {
             ((HttpServletResponse) response).sendError(UNAUTHORIZED.getStatusCode(), UNAUTHORIZED.getReasonPhrase());
         }
     }
 
     @Override
-    public void destroy() {}
+    public void destroy() {
+    }
 
-    private boolean isValidAuthorizationHeader(String authorization) {
-        return authorization!= null
+    private boolean isValidAuthorizationHeader(String authorization, ServletRequest request) {
```

Comment:
If we're only going to need the remote address is it worth just passing that through as a string? 

(After typing this I'm not so sure about it, the method signature would be confusing to understand as this method doesn't actually use the request but passes it on to the next. Happy for this to be done either way!)